test/Microsoft.ML.Tests/Scenarios/Api/CookbookSamples/CookbookSamples.cs

```diff
@@ -655,7 +655,7 @@ private void MixMatch(string dataPath)
             var binaryTrainer = mlContext.BinaryClassification.Trainers.AveragedPerceptron("Label", "Features");
 
             // Append the OVA learner to the pipeline.
-            dynamicPipe = dynamicPipe.Append(mlContext.MulticlassClassification.Trainers.OneVersusAll(binaryTrainer));
+            dynamicPipe = dynamicPipe.Append(mlContext.MulticlassClassification.Trainers.OneVersusAll(binaryTrainer, useProbabilities: false));
```

Comment:
Whenever you update the CookbookSamples.cs file, it usually means you need to update the [https://github.com/dotnet/machinelearning/blob/master/docs/code/MlNetCookBook.md](https://github.com/dotnet/machinelearning/blob/master/docs/code/MlNetCookBook.md) file as well.